configstack/module.go

```diff
@@ -128,12 +129,30 @@ func getTerragruntSourceForModule(modulePath string, moduleTerragruntConfig *con
 		return "", nil
 	}
 
-	matches := moduleUrlRegexp.FindStringSubmatch(moduleTerragruntConfig.Terraform.Source)
-	if len(matches) != 2 {
+	var moduleRelativePath string
```

Comment:
I suspect we'll need a method to separate the module path from the URL elsewhere, so would you mind extracting this into a new `getModulePathFromSourceUrl(sourceUrl string) (string, error)` method? That would also simplify the code a bit and remove the need for this `moduleRelativePath` variable, as you could `return` the relative path as soon as you found it.

Reply:
Good idea, I will work on that.